src/main/java/com/google/sps/data/User.java

```diff
@@ -14,12 +14,66 @@
 
 package com.google.sps.data;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.QueryResultList;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+
 public final class User {
   private String id;
   private boolean isMaintainer = false;
+  private String userEmail;
 
-  public User(String id, boolean isMaintainer) {
+  public User(String id, boolean isMaintainer, String userEmail) {
     this.id = id;
     this.isMaintainer = isMaintainer;
+    this.userEmail = userEmail;
+  }
+
+  public boolean getMaintainerStatus() {
```

Comment:
nit: Is there a different name you could use for this? I'm picturing a hypothetical use case:

myFn {
  if (user.getMaintainerStatus()) {
     // Is the user a maintainer, or was I able to successfully query the db / find out if they were logged in?
  }
}

What you you think about naming this "IsMaintainer()"?